### ISSUE TITLE
Usb dvb stick problem

### ISSUE THREAD

**Issue by dannyplomp** [jernejsk/OpenELEC-OPi2#69]
Hi Jernej,

I own an opipc board and i'm following your work for some time now, you do great work!
After the update today my mygica t230 usb dvb stick is not working anymore (in the june version it did worked). It does show up in lsusb but that is all. 

I installed your OE image from june again. And the usb stick worked like a charm.
With an clean install from your latest OE the problem still is there. :(

Maybe you can help me beat this problem. 
I have tried my other dvb stick(astrometa mn88472) but there are no firmware for it on your OE.
I hope you can add the mn88472 driver also ^_^

Thank you in advance!

Kind regards,
Danny

**Comment by jernejsk** [comment]
Linux 3.4.112 is pretty old, so it supports less DVBs. Sometime in the January (probably), backported media drivers (DVBs, IR, etc.) were added. But this broke some other DVB drivers, so it was lose - lose situation. Because those backported drivers caused some other issues and were removed from official OpenELEC, I decided to remove them, right after June images. However, if you provide me with a patch or source of working drivers on linux 3.4 for your DVB dongle then I will happily include them. This already happened with rtl2832 and dvbsky DVB drivers. I think I spend too much time on this already, so I won't actively search for drivers.

BTW: Mixing backported and native drivers is not possible without a ton of work and even then I wouldn't count on them, because I don't have any DVB dongle. Also media-build project (backported drivers) highly discourage use of backported drivers in production.

**Comment by dannyplomp** [comment]
Thank you for your help! I'm going to search for a patch or source working drivers. (I dont know how but i'm gonna newbie my way out of this haha) Thank you a lot!

Ps. I tried your test hdmi-cec version from a few weeks ago. The t-230 did also worked on that. i found the needed drivers for t230 in your last version. Too bad it stopped working.

Thank you for your help and understanding :)

**Comment by dannyplomp** [comment]
Do you need something like this?
(For astrometa mn 88472)

http://palosaari.fi/linux/v4l-dvb/firmware/MN88472/02/7253efc6959a1b9a36115bbc3b7eadc5/

Kind regards
Danny

**Comment by jernejsk** [comment]
That is only firmware (FW) - that gets loaded on the stick. What I need is a driver - code which runs in kernel and knows how to talk with the stick, like:
https://github.com/jernejsk/OpenELEC-OPi2/blob/openelec-7.0/projects/H3/patches/linux/linux-500-rtl2832.patch or
https://github.com/jernejsk/OpenELEC-OPi2/blob/openelec-7.0/projects/H3/patches/linux/linux-509-support-for-dvbsky-usb-receiver.patch

 FW part is already covered for that DVB stick (t230) on OpenELEC, but not driver. I took a quick look at driver for t230. At first glance nothing seems to suggest that it couldn't work on linux 3.4 if you would port it by hand. But this is definetly not easy.

**Comment by dannyplomp** [comment]
Thanks for the info. 
I dont want to be an pain in the ass with this so i will drop my t230 question.

I think found drivers for my other dvb stick MN88472 

https://kernel.googlesource.com/pub/scm/linux/kernel/git/mchehab/linux-media/+/media/v4.7-2/drivers/staging/media/mn88472/mn88472.c

And
http://www.spinics.net/lists/linux-media/msg83075.html

If i had more knowledge about this i would not waste your time with my questions ^^ 

I hope that is not to much work for you. :)
If i am on the wrong track please let me know

**Comment by jernejsk** [comment]
Yes, you found driver for your stick. However, it is for kernel 4.7 (latest stable) and it wouldn't work for linux 3.4. What you must found is backported driver. Another thing - DVB drivers for one DVB stick usually consist from 3 separate driver - frontend, tuner and interface. I'm not sure if mn88472 is 3 part or not (seems not). The easiest thing to do for you is IMO to clone my sources, reintroduce media_build package (revert commit which removed it) and build an image.

**Comment by dannyplomp** [comment]
Thanks!! I will try my best to make it work ^^
